debian/enterprise-kafka/Dockerfile.rpm

```diff
@@ -30,7 +30,7 @@ EXPOSE 9092
 
 RUN echo "===> installing confluent-support-metrics ..." \
     && yum -q -y update \
-    && yum install -y \
+    && yum install -y --nogpgcheck \
```

Comment:
Yeah, this would be great.  Same for switching between public repo urls and snapshot/private ones.